pkg/provider/bao/bao.go

```diff
@@ -158,8 +158,8 @@ func parsePathsToMap(paths []string) map[string]string {
 
 	for _, path := range paths {
 		split := strings.SplitN(path, "=", 2)
-		key, value := split[0], split[1]
-		baoEnviron[key] = value
+		originalKey, value := split[0], split[1]
+		baoEnviron[originalKey] = value
```

Comment:
In the future, I'd recommend refactors/fixes like this to at least be part of a separate commit. If something needs to be reverted, these changes would be reverted, too.